recipes/yaqc-qtpy/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "yaqc-qtpy" %}
+{% set version = "2021.12.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yaqc-qtpy-{{ version }}.tar.gz
+  sha256: 1feaa68c2e267cba9e589fd366ef162c069eb67f6ca90ee4b3e5d0d510bd9c16
+
+build:
+  script_env:
+    - QT_QPA_PLATFORM=offscreen
+  number: 0
+  noarch: python
```

```suggestion
  noarch: python
  entry_points:
    - yaqc-qtpy=yaqc_qtpy.__main__:main
```
I haven't tested it but I'd assume the error comes from the missing entry points.
Because upstream lists the entry point only in the `pyproject.toml`, Grayskull is probably not able to extract them which is (If you used it) why they weren't added. 

Ref: https://gitlab.com/yaq/yaqc-qtpy/-/blob/main/pyproject.toml#L33

---

Oops! Should have known that... thanks @BastianZim

---

No worries has happened to me as well. 😄